startup.py

```diff
@@ -188,21 +188,21 @@ def _find_software(self):
                 self.logger
             )
 
-        for install_paths in install_paths_dicts:
-            executable_version = self._map_version_year(install_paths["version"])
-            executable_path = install_paths["path"]
-            launcher_name = install_paths["_name"]
-            icon_file = self._icon_from_executable(launcher_name)
-
-            # Create The actual SoftwareVersions
-            sw_versions.append(
-                SoftwareVersion(
-                    executable_version,
-                    launcher_name,
-                    executable_path,
-                    icon_file,
+            for install_paths in install_paths_dicts:
+                executable_version = self._map_version_year(install_paths["version"])
+                executable_path = install_paths["path"]
+                launcher_name = install_paths["_name"]
+                icon_file = self._icon_from_executable(launcher_name)
+
+                # Create The actual SoftwareVersions
+                sw_versions.append(
+                    SoftwareVersion(
+                        executable_version,
```

Comment:
@rob-aitchison this hound error does not show up when I run the hook locally -- do you know what this is complaining about?

Reply:
Hey @staceyoue , I am not sure. I just ran a `black --check` from a prompt and it said it would leave it unchanged as well. I will see if there are any Hound log anomalies.